src/components/projects/projectId/attributes/attributeId/LLM/InputWithSlider.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
```

Comment:
this file looks like something that can be part of the react submodules

Reply:
moved